tests/storage-compat/populate_db/populate_db.py

```diff
@@ -0,0 +1,580 @@
+import requests
+import random
+import os
+
+QDRANT_HOST = os.environ.get("QDRANT_HOST", "localhost:6333")
+
+
+def drop_collection(name: str):
+    # cleanup collection if it exists
+    requests.delete(f"http://{QDRANT_HOST}/collections/{name}")
+
+
+def create_collection(name: str, quantization_config: dict = None):
+    # create collection with a lower `indexing_threshold_kb` to generate the HNSW index
+    response = requests.put(
+        f"http://{QDRANT_HOST}/collections/{name}",
+        headers={"Content-Type": "application/json"},
+        json={
+            "vectors": {"size": 50, "distance": "Dot"},
+            "optimizers_config": {
+                "default_segment_number": 2,
+                "indexing_threshold_kb": 10,
+            },
+            "quantization_config": quantization_config,
+        },
+    )
+    assert response.ok
+
+
+def create_payload_indexes(name: str):
+    # Create some payload indexes
+    response = requests.put(
+        f"http://{QDRANT_HOST}/collections/{name}/index",
+        json={"field_name": "city", "field_type": "keyword"},
+    )
+    assert response.ok
+
+    response = requests.put(
+        f"http://{QDRANT_HOST}/collections/{name}/index",
+        json={"field_name": "count", "field_type": "integer"},
+    )
+    assert response.ok
+
+
+def rand_vec(dims: int = 50):
```

Comment:
Let's generate more realistically sized vectors while we are at it.

I am proposing `256` instead to give product quantization some more work.